masterlist.yaml

```
@@ -514,6 +514,15 @@ plugins:
         subs: [ '"Script Improvements" version' ]
         condition: 'many("gr_ScriptImprovements.es(m|p)")'
 
+###### Environment, Landscape & Flora ######
+  - name: 'Better Landscapes Stonewood Pass (RP Edit).esp'
+    after:
+      - name: 'DD_Caldera_Expansion.esp'
+      - name: 'Walkers of Morrowind.esp'
+      - name: 'Foyada Mamaea Overhaul.ESP'
+      - name: 'DD_Caldera_Expansion.esp'
+      - name: 'Order_Of_Sanctity.ESP'
```

You only need the `name:` on files if you're adding a condition or display name.
Reordered alphabetically to make it easier to read.

```suggestion
      - 'DD_Caldera_Expansion.esp'
      - 'Foyada Mamaea Overhaul.ESP'
      - 'Order_Of_Sanctity.ESP'
      - 'Walkers of Morrowind.esp'
```